code/modules/power/supermatter/supermatter.dm

```diff
@@ -12,6 +12,13 @@
 #define OBJECT (LOWEST + 1)
 #define LOWEST (1)
 
+// length of returned list is our bool
+#define IS_CASCADING LAZYLEN
```

Comment:
this is cursed, don't do this

Reply:
the proc can't return two things at once in its current use case anyways, returning a list of reasons is completely useless